### PR TITLE
drop UI

### PR DESCRIPTION
**Pull request by wvchallenges**
First draft of dropping the UI component of the challenge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,159 +40,158 @@ In addition, the file name should be of the format `time-report-x.csv`,
 where `x` is the ID of the time report. For example, `time-report-42.csv` would
 represent a report with an ID of `42`.
 
-Our partner has guaranteed that:
+In addition, the file name of the time report should be of the format `time-report-x.csv`, where `x` is the ID of the time report. For example, `time-report-42.csv` would represent a report with an ID of `42`.
+
+You can assume that:
```
Comment (JBKahn) [inline README.md:44]: Report ids will always be integers?

File: README.md [hunk 2]
```diff
@@ -44,159 +40,158 @@ In addition, the file name should be of the format `time-report-x.csv`,
 where `x` is the ID of the time report. For example, `time-report-42.csv` would
 represent a report with an ID of `42`.
 
-Our partner has guaranteed that:
+In addition, the file name of the time report should be of the format `time-report-x.csv`, where `x` is the ID of the time report. For example, `time-report-42.csv` would represent a report with an ID of `42`.
```
Comment (JBKahn) [inline README.md:42]: We should update the playbook or rubric to assess what the submission does when there are 0 or more rows given we specify 0 as a valid CSV here.

File: README.md [hunk 3]
```diff
@@ -44,159 +40,158 @@ In addition, the file name should be of the format `time-report-x.csv`,
...
+
+   _NOTE:_ It is not the responsibility of the API to return html, as we will delegate the visual layout and redering to the front end. The expectation is that this API will only return JSON data.
+
+   - Return a JSON object `payrollReport`.
+   - `payrollReport` will have a single field, `employeeReports`, containing a list of objects with fields `employeeId`, `payPerdiod`, and `amountPaid`.
+   - The `payPeriod` field is an object containing a date interval that is roughly biweekly. Each month has two pay periods; the _first half_ is from the 1st to the 15th inclusive, and the _second half_ is from the 16th to the end of the month, inclusive. `payPeriod` will have two fields to represent this interval: `startDate` and `endDate`.
+   - Each employee should have a single object in `employeeReports` for each pay period that they have recorded hours worked. The `amountPaid` field should contain the sum of the hours worked in that pay period multiplied by the hourly rate for their job group.
+   - If an employee was not paid in a specific pay period, there should not be an object in `employeeReports` for that employee + pay period combination.
```
Comment (JBKahn) [inline README.md:71]: If the csv file contains a row for 0 hours for employee A, should they be present vs containing no records for employee B and having them be absent? Will we guarantee each employee in the CSV has at least one hour?

File: README.md [hunk 4]
```diff
@@ -44,159 +40,158 @@ In addition, the file name should be of the format `time-report-x.csv`,
...
-off one of your strengths.
+We've agreed to build an API with the following endpoints to serve HTTP requests:
+
+1. An endpoint for uploading a file.
+
+   - This file will conform to the CSV specifications outlined in the previous section.
+   - Upon upload, the timekeeping information within the file must be stored to a database for archival purposes.
+   - If an attempt is made to upload a file with the same report ID as a previously uploaded file, this upload should fail with an error message indicating that this is not allowed.
+
+1. An endpoint for retrieving a payroll report structured in the following way:
```
Comment (JBKahn) [inline README.md:63]: Should we consider an optional start/end date or pay period param for filtering this endpoint? It slightly increases the complexity but I think gives us something to look at during the code evaluation.

Comment (warnockf) [reply README.md:63]: Leaving this out for now. I'll bring it up with the working group in our next meeting.

File: README.md [hunk 5]
```diff
@@ -41,162 +37,160 @@ Each individual CSV file is known as a "time report", and will contain:
...
+
+   _NOTE:_ It is not the responsibility of the API to return html, as we will delegate the visual layout and redering to the front end. The expectation is that this API will only return JSON data.
```
Comment (dhoffman34) [inline README.md:65]: nit: should be "HTML"

**Pull request merged**